core/templates/dev/head/forms/formBuilder.js

```diff
@@ -627,6 +641,10 @@ oppia.factory('rteHelperService', [
           el.attr(
             $filter('camelCaseToHyphens')(attrName) + '-with-value',
             oppiaHtmlEscaper.objToEscapedJson(customizationArgsDict[attrName]));
+          if (componentDefn.name === 'image' && attrName === 'width') {
```

Comment:
I added this so that the preview image shows the right width on the editor too. Without that it would show the original image in the RTE, but on clicking "save content" the modified image shows on the card.